plugins/lsp/language-servers/_mk-lsp.nix

```
@@ -37,7 +37,7 @@ in
 {
   meta.nixvimInfo = {
     # TODO: description
-    url = args.url or opts.package.default.meta.homepage or null;
+    url = args.url or (builtins.tryEval opts.package.default).value.meta.homepage or null;
```

Nit: we should probably comment _why_ were using `tryEval`


```suggestion
    # The package default can throw when undefined, so use tryEval
    # See mkUnpackagedOption
    url = args.url or (builtins.tryEval opts.package.default).value.meta.homepage or null;
```